Makefile

```diff
@@ -1,21 +1,72 @@
-default: build run 
+SHELL := /bin/bash
+DATABASE_VOLUME := food
 
+# List of all targets which are meant to be run by the user (should have a description)
+ALL_TARGETS := $(shell sed -n 's/^## //p' ${MAKEFILE_LIST} | awk -F ':' '{print $$1}')
+
+PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
```
PROJECT_DIR := $(CURDIR)
```
or even better, add `notdir` there itself.

Reply:
wont work if i run make from any other directory

Reply:
Hmm, then this is better.